unittests/CMakeLists.txt

```diff
@@ -107,6 +107,12 @@ else()
       target_include_directories(gtest BEFORE PRIVATE ${LLVM_INCLUDE_DIRS})
       # we cannot disable gtest_main, but will not use it later
       target_include_directories(gtest_main BEFORE PRIVATE ${LLVM_INCLUDE_DIRS})
+    endif()
+
+    # try using system installation of GTest instead
+    find_package(GTest CONFIG QUIET)
```

Comment:
@lzaoral looking at this again, is there any reason why you want to restrict `find_package` to Config Mode, and exclude the use of https://cmake.org/cmake/help/latest/module/FindGTest.html?

Reply:
Not really, I just didn't know that CMake has it's own `GTest` module at the time when I wrote the original patch. Addressed in 2a052120.